src/System.Data.SqlClient/src/System/Data/SqlClient/SqlException.cs

```diff
@@ -11,25 +11,47 @@
 using System.Data.Common;
 using System.Diagnostics;
 using System.Globalization;
+using System.Runtime.Serialization;
 using System.Text; // StringBuilder
 
 namespace System.Data.SqlClient
 {
+    [Serializable]
     public sealed class SqlException : System.Data.Common.DbException
```

Comment:
> Where should SqlEXception.GetObjectData get tested then? It seems like you've added tests for some but not others so I'm trying to figure out how you chose. I get that [Serializable] is probably not worth testing as it's just memberwise. 

In general I've tried to add tests in most places, and in particular anywhere there's a custom implementation (e.g. ISerializable), though it's still valuable for [Serializable] because trying to serialize something that's just [Serializable] could still fail if it's not actually serializable, e.g. contains fields of types that can't be serialized and aren't marked as [NonSerialized].  That said, for this PR, I've stayed away from upgrading projects to netstandard1.7, as there's just too much work involved to do it for essentially all of the projects we have while also doing these serialization changes.  And the SqlClient test project is still only netstandard1.3.